runtime/doc/treesitter.txt

```diff
@@ -647,6 +647,18 @@ attribute: >query
     ((super_important_node) @superimportant (#set! priority 105))
 <
 
+                                               *treesitter-highlight-commentstring*
+Treesitter highlighting supports finer-grained 'commentstring's, used by the
+built-in |commenting| plugin. When the cursor is within a node with the
+`commentstring` metadata property set, the comment marker will be set to that
+string. This is useful for languages like `JSX` that have different comment
```

Comment:
I personally prefer to avoid using specific languages when documenting language-agnostic functionality in favor of something like "take a look at the queries at this link".

But documentation is something where @clason has more authority.

Reply:
concrete examples are good, though linking to actual queries is also good (can have both)

Reply:
Maybe I can come back here and add a link once `nvim-treesitter` has some examples?